plugins/jobs/src/js/components/form/helpers/JobFormData.ts

```diff
@@ -26,7 +26,7 @@ export interface JobRun<Env, Secrets> {
   secrets?: Secrets;
 }
 
-export type JobFormData = Job<ArrayLabels, EnvModel, SecretModel>;
+export type JobSpecData = Job<ArrayLabels, EnvModel, JobSecretExposure>;
```

Comment:
I think we will need `JobSecretExposure[]` here

Reply:
😭  of course